ui/src/app/my-workspace/my-workspace.component.html

```diff
@@ -0,0 +1 @@
+<p>my-workspace works!</p>
```

Comment:
Is this actual code for the feature? Leftover sketch code?

Reply:
This file was removed